requirements.txt

```
@@ -1,5 +1,5 @@
-uvicorn
-fastapi
-pyyaml
-aiosmtplib
-python-dotenv
+uvicorn==0.23.2
```

I recently read something about not upper-bounding Python requirements. Shall we do that?

https://iscinumpy.dev/post/bound-version-constraints/

```suggestion
uvicorn>=0.23.2
```